cli/cartridge/extra/001_make_cmd_public.patch

```diff
@@ -0,0 +1,212 @@
+diff --git a/cli/commands/admin.go b/cli/commands/admin.go
+index 5a37a75..77fdb4e 100644
+--- a/cli/commands/admin.go
++++ b/cli/commands/admin.go
+@@ -9,6 +9,8 @@ import (
+ 	"github.com/tarantool/cartridge-cli/cli/admin"
+ )
+ 
++var CartridgeCliAdmin *cobra.Command
++
+ func init() {
+ 	var adminCmd = &cobra.Command{
+ 		Use:   "admin [ADMIN_FUNC_NAME]",
+@@ -27,6 +29,7 @@ Otherwise, first available socket from all <run-dir>/<app-name>.*.control is use
+ 		DisableFlagParsing: true,
+ 	}
+ 
++	CartridgeCliAdmin = adminCmd
+ 	rootCmd.AddCommand(adminCmd)
+ 
+ 	// FLAGS are parsed in runAdminCommand
+diff --git a/cli/commands/bench.go b/cli/commands/bench.go
+index 8e408a6..06d85f0 100644
+--- a/cli/commands/bench.go
++++ b/cli/commands/bench.go
+@@ -6,6 +6,8 @@ import (
+ 	"github.com/tarantool/cartridge-cli/cli/bench"
+ )
+ 
++var CartridgeCliBench *cobra.Command
++
+ func init() {
+ 	var benchCmd = &cobra.Command{
+ 		Use:   "bench",
+@@ -17,6 +19,7 @@ func init() {
+ 			}
+ 		},
+ 	}
++	CartridgeCliBench = benchCmd
+ 	rootCmd.AddCommand(benchCmd)
+ 
+ 	configureFlags(benchCmd)
+diff --git a/cli/commands/build.go b/cli/commands/build.go
+index 5d444d3..127ea43 100644
+--- a/cli/commands/build.go
++++ b/cli/commands/build.go
+@@ -7,6 +7,8 @@ import (
+ 	"github.com/tarantool/cartridge-cli/cli/build"
+ )
+ 
++var CartridgeCliBuild *cobra.Command
++
+ func init() {
+ 	var buildCmd = &cobra.Command{
+ 		Use:   "build [PATH]",
+@@ -21,6 +23,7 @@ func init() {
+ 		},
+ 	}
+ 
++	CartridgeCliBuild = buildCmd
+ 	rootCmd.AddCommand(buildCmd)
+ 
+ 	// FLAGS
+diff --git a/cli/commands/clean.go b/cli/commands/clean.go
+index 15a3dc3..42e1879 100644
+--- a/cli/commands/clean.go
++++ b/cli/commands/clean.go
+@@ -9,6 +9,8 @@ import (
+ 	"github.com/tarantool/cartridge-cli/cli/running"
+ )
+ 
++var CartridgeCliClean *cobra.Command
++
+ func init() {
+ 	var cleanCmd = &cobra.Command{
+ 		Use:   "clean [INSTANCE_NAME...]",
+@@ -23,6 +25,7 @@ func init() {
+ 		ValidArgsFunction: ShellCompRunningInstances,
+ 	}
+ 
++	CartridgeCliClean = cleanCmd
+ 	rootCmd.AddCommand(cleanCmd)
+ 
+ 	// FLAGS
+diff --git a/cli/commands/connect.go b/cli/commands/connect.go
+index 7eebcd6..26c8c42 100644
+--- a/cli/commands/connect.go
++++ b/cli/commands/connect.go
+@@ -6,6 +6,11 @@ import (
+ 	"github.com/tarantool/cartridge-cli/cli/connect"
+ )
+ 
++var (
```

Comment:
AFAIU commands `Connect` and `Enter` we do not export, why do we create these variables?

Reply:
Fixed